integration-tests/gatsby-source-wordpress/__tests__/__snapshots__/index.js.snap

```diff
@@ -6259,9 +6241,62 @@ Array [
       "status",
       "template",
       "uri",
+      "parent",
+      "children",
+      "internal",
     ],
     "name": "WpNodeWithFeaturedImage",
   },
+  Object {
+    "fields": Array [
+      "totalCount",
+      "edges",
+      "nodes",
+      "pageInfo",
+      "distinct",
+      "max",
+      "min",
+      "sum",
+      "group",
+    ],
+    "name": "WpNodeWithFeaturedImageConnection",
+  },
+  Object {
+    "fields": Array [
+      "next",
+      "node",
+      "previous",
+    ],
+    "name": "WpNodeWithFeaturedImageEdge",
+  },
+  Object {
+    "fields": null,
+    "name": "WpNodeWithFeaturedImageFieldSelector",
+  },
+  Object {
+    "fields": null,
+    "name": "WpNodeWithFeaturedImageFilterInput",
+  },
+  Object {
+    "fields": Array [
+      "totalCount",
+      "edges",
+      "nodes",
+      "pageInfo",
+      "distinct",
+      "max",
+      "min",
+      "sum",
+      "group",
+      "field",
+      "fieldValue",
+    ],
+    "name": "WpNodeWithFeaturedImageGroupConnection",
+  },
+  Object {
+    "fields": null,
+    "name": "WpNodeWithFeaturedImageSortInput",
+  },
```

Comment:
Some wpgql types that were previously ignored now need to be included. Basically all interface types that implement Node need to be included now